Yank/multistate/multistatereporter.py

```diff
@@ -54,7 +54,7 @@
 # MULTISTATE SAMPLER REPORTER
 # ==============================================================================
 
-MISSING_VALUE = np.NaN # value for populating missing/masked values
+MISSING_VALUE = np.inf # value for populating missing values
```

Comment:
I'm not sure `inf` is a good idea here since we can have real states with `inf` energy (think decoupled atom on top of normal atom evaluated in fully coupled state)